src/requests/team.ts

```diff
@@ -76,6 +76,10 @@ export class TeamClient {
     return getIntegrations({ client: this.client, ...args });
   }
 
+  async getTeamSettings(slug: string): Promise<TeamSettings> {
+    return getTeamSettings(this.client, slug);
+  }
```

Comment:
I don't think these types and return values are correctly handled.

This function looks like it actually returns `{ settings: TeamSettings }`, but I think we want to just return a single instance of `TeamSettings` (as per your function type signature here).